src/components/hv-view/types.js

```diff
@@ -12,6 +12,7 @@ import type { StyleSheet } from 'react-native/Libraries/StyleSheet/StyleSheetTyp
 
 export type InternalProps = {|
   accessibilityLabel?: ?string,
+  behavior?: ?string,
```

Comment:
perhaps restrict to the allowed values, i.e.

```suggestion
  behavior?: 'position',
```